src/modules/launcher/Plugins/Microsoft.Plugin.Registry/plugin.json

```diff
@@ -0,0 +1,12 @@
+{
+  "ID": "303417D927BF4C97BCFFC78A123BE0C8",
+  "ActionKeyword": "*",
```

Comment:
This should only be invoked by a action keyword, not everything.

Reply:
fixed, it is now `"ActionKeywords": [ "HK", "hk", "Hk", "hK" ],` we need a four combinations, because it is case-sensitive